From 9b65f0a17dd75a75d69b23ad8ab51ea307783791 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Wed, 18 May 2011 12:46:20 +0800
Subject: [PATCH 296/678] ENGR00143613 uboot: update the android ramdisk load
 address

update the ramdisk load address due to android kernel size enlarge.
the ramdisk memory load address is 5MB offset to kernel.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 include/configs/mx51_bbg_android.h |    2 +-
 include/configs/mx53_evk_android.h |    2 +-
 include/configs/mx53_smd_android.h |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/configs/mx51_bbg_android.h b/include/configs/mx51_bbg_android.h
index 6206678..b583719 100644
--- a/include/configs/mx51_bbg_android.h
+++ b/include/configs/mx51_bbg_android.h
@@ -167,7 +167,7 @@
 		"uboot_addr=0xa0000000\0"				\
 		"uboot=u-boot.bin\0"			\
 		"kernel=uImage\0"				\
-		"rd_loadaddr=0x90C00000\0"	\
+		"rd_loadaddr=0x90D00000\0"	\
 		"nfsroot=/opt/eldk/arm\0"				\
 		"bootargs_base=setenv bootargs console=ttymxc0,115200 gpu_memory=16M\0"\
 		"bootargs_nfs=setenv bootargs ${bootargs} root=/dev/nfs " \
diff --git a/include/configs/mx53_evk_android.h b/include/configs/mx53_evk_android.h
index b27bffb..bdb2a95 100644
--- a/include/configs/mx53_evk_android.h
+++ b/include/configs/mx53_evk_android.h
@@ -164,7 +164,7 @@
 		"uboot=u-boot.bin\0"					\
 		"kernel=uImage\0"					\
 		"loadaddr=0x70800000\0"					\
-		"rd_loadaddr=0x70C00000\0"				\
+		"rd_loadaddr=0x70D00000\0"				\
 		"nfsroot=/opt/eldk/arm\0"				\
 		"bootargs_base=setenv bootargs ${bootargs} "		\
 		"console=ttymxc0,115200 gpu_memory=16M\0"		\
diff --git a/include/configs/mx53_smd_android.h b/include/configs/mx53_smd_android.h
index a9d06f4..04fc808 100644
--- a/include/configs/mx53_smd_android.h
+++ b/include/configs/mx53_smd_android.h
@@ -160,7 +160,7 @@
 		"uboot=u-boot.bin\0"                                    \
 		"kernel=uImage\0"                                       \
 		"loadaddr=0x70800000\0"                                 \
-		"rd_loadaddr=0x70C00000\0"                              \
+		"rd_loadaddr=0x70D00000\0"                              \
 		"nfsroot=/opt/eldk/arm\0"                               \
 		"bootargs_base=setenv bootargs console=ttymxc0,115200\0"\
 		"bootargs_nfs=setenv bootargs ${bootargs} root=/dev/nfs " \
-- 
1.7.7.4

